src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -236,7 +236,7 @@ va.hist.nGreater == va.nGreater
           |va.AC = gs.map(g => g.GT.oneHotAlleles(v)).sum(),
           |va.GC = gs.map(g => g.GT.oneHotGenotype(v)).sum(),
           |va.AN = gs.filter(g => isDefined(g.GT)).count() * 2""".stripMargin)
-      .annotateVariantsExpr("va.AF = va.AC / va.AN")
+      .annotateVariantsExpr("va.AF = va.AC.map(x => x.toFloat64()) / va.AN")
```

Comment:
`va.AN.toFloat64()` achieves the same ends right?

Reply:
yep

Reply:
oh, that's way simpler